.github/workflows/go.yml

```diff
@@ -2,17 +2,12 @@ name: Go
 
 on:
   push:
-    branches: [ main ]
```

Comment:
I'd undo this, so we still only run these tests on pushes to main and PRs to main. Otherwise you get CI running in forks when folks push to a branch in their fork.

Reply:
Done.